### PR TITLE
Tutorial for mesh distortion in Blender Python

### PR DESCRIPTION
**Pull request by mabelzhang**
# 🎉 New feature

Blender Python script and tutorial for programmatically distorting 3D meshes.

Upstreaming from [Project DAVE](https://github.com/Field-Robotics-Lab/dave).

## Summary
See tutorial Markdown file.

## Test it

Blender 2.92 (newer might work too) is required - I'm not sure who's the best to review this?

Follow instructions in tutorial.
Check that the tutorial renders correctly by compiling and then navigating in a browser to `ign/build/ignition-gazebo7/doxygen/html/blender_distort_meshes.html`

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [x] Added example and/or tutorial
- [x] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Comment by mayman99** [comment]
Works as expected!

imho I would personally prefer if we didn't have to use scripting inside blender or blender GUI at all to produce results, like instead of writing 
```python
   file_path = '/path/to/file.dae'
   object_prefix = 'Cube'
   distort_extent = 0.1  # float in range [0, 1]
   method = 'deform'  # method of distortion
   sys.argv = ['distort_mesh.py', file_path, object_prefix, distort_extent, method]
   exec(open('/path/to/distort_mesh.py').read());
```
inside the blender console, having them as a part of the script, plus a `bpy import object` on top.
Then the user will have to only run something like `./blender -b -P <distort_script> args...` from the terminal.

**Comment by mayman99** [comment]
@mabelzhang  friendly ping :)

**Review by AzulRadio — COMMENTED** [review]
The script looks good. 

Tested with Blender 2.93 LTS and 3.0.0 on Windows. The results are slightly different. Are these results still considered good? 

2.93 LTS with 0.04 distort
2 93_0 04 distort

3.0.0 with 0.2 distortion (default distortion in the tutorial)
3 0 0_0 2 distort

File: tutorials/blender_distort_meshes.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+At the top of Blender GUI, go to the Scripting tab.
+All commands will be executed in the Console panel in the middle-left of the
+screen.
+
+Define the full path to the model file and the mesh name, which you found above.
+For example, the `Coke` model:
+```
+file_path = '/path/to/models/Coke/meshes/coke.obj'
+object_prefix = 'LPCoke_Cube'
```
[the remark this answers is not in the document]
Comment (mabelzhang) [reply tutorials/blender_distort_meshes.md:115]: Thank you! I updated the README and added this a bit higher in the tutorial. Please check if it makes it clear 7dc438140

**Comment by mabelzhang** [comment]
> The results are slightly different. Are these results still considered good?

Thank you for catching that. Yeah I adjusted the `VERT_RAND_MIN` and `VERT_RAND_MAX` and forgot to update the README. It's a bit tricky to set for the general case and will need to be tuned for each model, I think.
d5acd69d7 adds that documentation.

**Comment by mabelzhang** [comment]
@mayman99 Sorry for the long delay, and thank you for pressing charge. We have an active budget for the supporting project again, so I can spend time on this again :)

Ready for another pass.

**Review by mayman99 — APPROVED** [review]
Thanks for adding the command line usage!
lgtm :+1:

**Pull request merged**